install/samples/lazyload/easy_uninstall_lazyload.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/bash
+if [[ "$#" -eq 0 ]]; then
+  tag_or_commit=$(curl -s https://api.github.com/repos/slime-io/slime/tags | grep 'name' | cut -d\" -f4 | head -1)
+  echo "No specified tag or commit. Use latest tag: $tag_or_commit."
+else
+  tag_or_commit=$1
+  echo "Use specified tag or commit: $tag_or_commit"
+fi
+
+crds_url="https://raw.githubusercontent.com/slime-io/slime/$tag_or_commit/install/init/crds.yaml"
+deployment_slimeboot_url="https://raw.githubusercontent.com/slime-io/slime/$tag_or_commit/install/init/deployment_slime-boot.yaml"
+slimeboot_lazyload_url="https://raw.githubusercontent.com/slime-io/slime/$tag_or_commit/install/samples/lazyload/slimeboot_lazyload.yaml"
+
 for i in $(kubectl get ns);do kubectl delete servicefence -n $i --all;done
```

Comment:
--all-namespaces ?

Reply:
--all means delete all svf in single namespace

Reply:
https://stackoverflow.com/questions/33509194/command-to-delete-all-pods-in-all-kubernetes-namespaces